src/entities/Votes/types.ts

```diff
@@ -21,6 +21,9 @@ export type VotedProposal = SnapshotVote & {
     proposal_id: string
     type: ProposalType
     status: ProposalStatus
+    author: string
+    finish_at: number
+    scores: number[]
```

Comment:
I'm wondering if we really need this `scores`, because the `SnapshotVote` already has the proposal with the scores inside it.

This part of the VotedProposal should only add that info that we don't store in snapshot.

You could also ask for the `author` in the same request to graphql.

Reply:
By this I am just saying that the properties are not optional, they are included

<img width="560" alt="image" src="https://user-images.githubusercontent.com/45410089/197770712-569df541-99a8-4c73-9608-29df88e62d56.png">